compiler/model/metamodel.ts

```diff
@@ -116,6 +116,13 @@ export class LiteralValue {
   value: string | number | boolean
 }
 
+/**
+ * The absence of any type. This is commonly used in APIs that returns an empty body.
+ */
+export class VoidValue {
+  kind: 'void_value'
+}
+
```

Comment:
Late review of this PR... IMHO we should remove this newly introduced `VoidValue`.

`void` is not a value, it's the absence of value (which is different from `null` and `undefined` that _are_ values). As such a property cannot be of type void, but adding it to the `ValueOf` enumeration makes it possible.

We already have a simpler means to represent requests and responses with an empty body (or more exactly no body, as an empty body could be `{}`): the `body` property is optional: `body?: ValueBody | PropertiesBody`. If it's not specified, then the request or response has no body.

Or is was this introduced so that body-less requests and responses are explicitly represented in the spec? In that case we should make the body property a required field and add a 3rd alternative: `body: ValueBody | PropertiesBody | VoidBody`